src/test/scala/org/broadinstitute/dsde/workbench/leonardo/TestSupport.scala

```diff
@@ -0,0 +1,11 @@
+package org.broadinstitute.dsde.workbench.leonardo
+
+import scala.concurrent.{Await, Awaitable}
+import scala.concurrent.duration.Duration
+
+/**
+  * Created by dvoet on 6/27/17.
+  */
+trait TestSupport {
```

Comment:
how does that work if i want to run future1, then future2, then future3, and wait on all of them in sequence? does that look like a sequence of `whenReady()` calls?